packages/docs/docs/concepts/progressive-loading/stackProgressive.md

```diff
@@ -0,0 +1,86 @@
+---
+id: stackProgressive
+---
+
+# Stack Progressive Loading
+
+Images for the stack viewport can be loaded with a lower resolution/lossy
+version first, followed by increasingly higher resolutions, and finally
+the final version being a lossless representation.
+
+For HTJ2K, this is done automatically when the image is encoded in progressive
+resolution order by using a streaming reader that returns lower resolution versions
+of the image as they are available.
+
+For other image types, a separate lower resolution/lossy version is required.
+The Static DICOMweb toolkit includes some options to create such images.
+
+# Performance
+
+In general, about 1/16-1/10th of the image is retrieved for the lossy/first
+version of the image. This results in about a 10x speed improvement to first
+images. It is affected fairly strongly by overall image size, network performance
+and compression ratios.
+
+| Type             | Network | First Render | Final Render |
+| ---------------- | ------- | ------------ | ------------ |
+| JLS              | 4g      |              | 7188 ms      |
+| JLS Reduced      | 4g      | 509 ms       | 7359 ms      |
+| HTJ2K            | 4g      | 128 ms       | 6878 ms      |
+| HTJ2K Byte Range | 4g      | 126 ms       | 6171 ms      |
+
+- JLS Reduced uses 1/16 size JLS 'thumbnails'
```

Comment:
Please avoid usage of thumbnails, is this really well known in the imaging community? that by thumbnails we mean subresolution?

Reply:
Lossy can be sub resolution or might be full resolution but lossy pixelData

Partial is ....

A lossy image can be partial which means even less data. 

As discussed ideally we need one